deployments/fargate/agent.yml

```diff
@@ -0,0 +1,65 @@
+# This agent config file is designed for use within an ECS cluster.  The agent
+# should be run as a Daemon service with the host EC2 instance's filesystem
+# mounted in at /hostfs and the docker socket mounted in at
+# /var/run/docker.sock.
+signalFxAccessToken: {"#from": "env:ACCESS_TOKEN"}
+ingestUrl: {"#from": "env:INGEST_URL", optional: true}
+apiUrl: {"#from": "env:API_URL", optional: true}
+
+intervalSeconds: {"#from": "env:INTERVAL_SECONDS", default: 10}
+
+logging:
+  level: {"#from": "env:LOG_LEVEL", default: "info"}
+
+etcPath: /hostfs/etc
+procPath: /hostfs/proc
+runPath: /hostfs/run
+varPath: /hostfs/var
+sysPath: /hostfs/sys
+
+# observers are what discover running services in the environment
+observers:
+    # Enable the docker observer to discover other ECS tasks running on the
+    # same EC2 instance.
+  - type: ecs
+    # ECS tasks running the awsvpc network mode don't actually expose and IP
+    # address on the docker inspect API so we need to use the hostname instead.
+    labelsToDimensions: &labelMap
+      # We map the cluster label to ClusterName to match what our AWS
+      # CloudWatch integration uses.
+      com.amazonaws.ecs.cluster: ClusterName
```

Comment:
We don't need these since the task metadata is gotten from the ECS Metadata agent.